Quizzler-iOS13/Model/QuizManager.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  QuizManager.swift
+//  Quizzler-iOS13
+//
+//  Created by Ronald Ngounou on 4/18/24.
+//  Copyright © 2024 The App Brewery. All rights reserved.
+//
+
+import Foundation
+
+class QuizManager{
```

Comment:
it's better to add `final` to your classes by default
`final class QuizManager {`

Reply:
Thank you!